src/lib/axiosConfig.ts

```diff
@@ -2,10 +2,10 @@ import axios from 'axios'
 import { store } from '../context/redux/store'
 import { getClientResponse } from '../context/redux/user'
 
-const localHost = process.env.REACT_APP_BACKEND_HOST
+const backendHost = process.env.REACT_APP_BACKEND_HOST
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/URL/host
pod zmienną REACT_APP_BACKEND_HOST się kryje wiecej niż host

Reply:
jakby apka łączyła się z więcej niż 1 backendem to zmienne by się nazywały backend1 backend2 backend3?